stock_picking_production_lot_manual_selector/models/stock_move_line.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    use_manual_lots = fields.Boolean(
+        related="picking_id.use_manual_lots",
+    )
+
+    manual_production_lot_id = fields.Many2one(
+        "stock.production.lot",
+        "Lot/Serial Number (manual)",
+        domain="[('product_id', '=', product_id), ('company_id', '=', company_id)]",
```

Comment:
By default, there is a check on locations if there is an incompatibility at confirmation.
```
Serial number (0123456789) is not located in A, but is located in location(s): B.

Please correct this to prevent inconsistent data.
```

Reply:
To check with the customer, often they ask the SN to be filtered so as not to see SN which have already been delivered to a customer when working on a Stock -> Customer  picking.